compose/composer.go

```diff
@@ -63,17 +95,21 @@ func ndjsonWrapObj(key string, buf []byte) []byte {
 	return buff.Bytes()
 }
 
-func multiply(key []byte, value []byte, times int) []byte {
-	var ret bytes.Buffer
-	var c int
-	ret.Write(key)
-	ret.WriteString(":[")
-	ret.Write(value)
-	for c < times-1 {
-		ret.WriteByte(',')
-		ret.Write(value)
-		c++
+func stacktrace(n int) []byte {
+	l := len(StacktraceFrames)
+	var buf bytes.Buffer
+	buf.Write([]byte(`"stacktrace": [`))
+	for i := 0; i < n; i++ {
+		if i > 0 {
+			buf.WriteByte(',')
+
+		}
+		randFrame := StacktraceFrames[rand.Intn(l)]
+		fr := make([]byte, len(randFrame))
+		copy(fr, randFrame)
+		buf.Write(fr)
 	}
-	ret.WriteString("],")
-	return ret.Bytes()
+	buf.WriteString("],")
+	ioutil.WriteFile("/tmp/sf.json", buf.Bytes(), 0644)
```

Comment:
why write to file?

Reply:
As @graphaelli pointed out, it is a leftover https://github.com/elastic/hey-apm/pull/64#discussion_r230445478